lisa/testsuite.py

```diff
@@ -237,10 +237,26 @@ class TestCaseRequirement:
     os_type: Optional[search_space.SetSpace[Type[OperatingSystem]]] = None
 
 
-def simple_requirement(
+def _construct_node(
+    node: Optional[schema.NodeSpace] = None,
     min_count: int = 1,
     min_nic_count: int = 1,
-    node: Optional[schema.NodeSpace] = None,
+    min_data_disk_count: int = 0,
+    data_disk_caching_type: str = constants.DATADISK_CACHING_TYPE_NONE,
+    data_disk_iops: int = 120,
+) -> schema.NodeSpace:
+    if node is None:
```

Comment:
After the logic is in simple_requirement, the `if` clause is not needed.

Reply:
fix